<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grov::main_window::Viewer</class>
 <widget class="QWidget" name="grov::main_window::Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>660</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="viewer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="grov::main_window::viewer::Feeds_view" name="feeds_view"/>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWebView" name="item_view">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="controls_box" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>152</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="star_check_box">
            <property name="toolTip">
             <string>Star this item &lt;Ctrl-S&gt;</string>
            </property>
            <property name="text">
             <string>Star</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>grov::main_window::viewer::Feeds_view</class>
   <extends>QTreeView</extends>
   <header location="global">grov/main_window/viewer/feeds_view.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
